app/src/main/res/values/strings.xml

```diff
@@ -138,4 +138,6 @@
     <string name="end_hour">End hour:</string>
     <string name="price">Price:</string>
     <string name="update_success">Update Success</string>
+    <string name="price_is_empty">Price is empty</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.price_is_empty` appears to be unused
 Unused resources make applications larger and slow down builds.